plugins/headlamp-plugin/README.md

```diff
@@ -11,18 +11,20 @@ This package is published to the npm package index separately from Headlamp.
 
 ```
 headlamp-plugin --help
-headlamp-plugin create <name>     Create a new plugin folder with base code
-headlamp-plugin build <package>   Build the plugin, or folder of
-                                  plugins. <package> defaults to
-                                  current working directory.
-headlamp-plugin start             Watch for changes and build the plugin
-headlamp-plugin upgrade <package> Upgrade the plugin to latest headlamp-plugin.
-                                  Audits, formats, lints and type checks.
-                                  <package> defaults to current working
-                                  directory. Can also be a folder of packages.
-headlamp-plugin extract           Copies folders of packages from plug
-<pluginPackages> <outputPlugins>  inPackages/packageName/dist/main.js
-                                  to outputPlugins/packageName/main.js
+headlamp-plugin create <name>       Create a new plugin folder with base code.
+headlamp-plugin build <package>     Build the plugin, or folder of
+                                    plugins. <package> defaults to
+                                    current working directory.
+headlamp-plugin start               Watch for changes and build the plugin.
+headlamp-plugin upgrade <package>   Upgrade the plugin to latest headlamp-plugin.
+                                    Audits, formats, lints and type checks.
+                                    <package> defaults to current working
+                                    directory. Can also be a folder of packages.
```

Comment:
Need updated usage docs from headlamp-plugin command. Format/lint/etc are missing and not updated for multiple packages.